docs/concepts/resource-limits-and-performance-optimization.md

```diff
@@ -99,8 +108,46 @@ While the resources limits on CPU and memory usage, crash tolerance, UI responsi
+### Performance improvements with the host-specific APIs
+
+The performance tips in [Using the host-specific API model](../develop/host-specific-api-model.md) provide guidance when using the host-specific APIs for Excel, OneNote, Visio, and Word. In summary, you should:
+
+- [Only load necessary properties](../develop/host-specific-api-model.md#calling-load-without-parameters-not-recommended).
+- [Minimize the number of sync() calls](../develop/host-specific-api-model.md#performance-tip-minimize-the-number-of-sync-calls). Read [Avoid using the context.sync method in loops](correlated-objects-pattern.md) for further information on how to avoid `sync` calls in your code.
+- [Minimize the number of proxy objects created](../develop/host-specific-api-model.md#performance-tip-minimize-the-number-of-proxy-objects-created). You can also untrack proxy objects, as described below.
+
+#### Untrack unneeded proxy objects
+
+[Proxy objects](../develop/host-specific-api-model.md#proxy-objects) persist in memory until `RequestContext.sync()` is called. Large batch operations may generate a lot of proxy objects that are only needed once by the add-in and can be released from memory before the batch executes.
+
+The `untrack()` method releases the object from memory. This method is implemented on many host-specific API proxy objects. Calling `untrack()` after your add-in is done with the object should yield a noticeable performance benefit when using large numbers of proxy objects.
+
+> [!NOTE]
+> `Range.untrack()` is a shortcut for [ClientRequestContext.trackedObjects.remove(thisRange)](/javascript/api/office/officeextension.trackedobjects#remove-object-). Any proxy object can be untracked by removing it from the tracked objects list in the context.
+
+The following Excel code sample fills a selected range with data, one cell at a time. After the value is added to the cell, the range representing that cell is untracked. Run this code with a selected range of 10,000 to 20,000 cells, first with the `cell.untrack()` line, and then without it. You should notice the code runs faster with the `cell.untrack()` line than without it. You may also notice a quicker response time afterwards, since the cleanup step takes less time.
+
+```js
+Excel.run(async (context) => {
+    var largeRange = context.workbook.getSelectedRange();
+    largeRange.load(["rowCount", "columnCount"]);
+    await context.sync();
+
+    for (var i = 0; i < largeRange.rowCount; i++) {
+        for (var j = 0; j < largeRange.columnCount; j++) {
+            var cell = largeRange.getCell(i, j);
+            cell.values = [[i *j]];
+
+            // Call untrack() to release the range from memory.
+            cell.untrack();
+        }
+    }
+
+    await context.sync();
+});
+```
 
+Note that needing to untrack objects only becomes important when you're dealing with thousands of them. Most add-ins will not need to manage proxy object tracking.
```

Comment:
```suggestion
Note that needing to untrack objects only becomes important when you're dealing with a large number of them. Most add-ins will not need to manage proxy object tracking.
```

Reply:
Hmm, I sort of wanted to be more specific here since the perception of a "large number" might be different. Like, we don't want customers worrying about tracking/untracking objects if they only have 100 of them.   Is the concern that the performance boundaries will change over time?

Also, FWIW, the original section listed "10,000 to 20,000 ranges" for the limit of needing `untrack`. I changed that to a more general number since ranges might be different sizes than other common objects in other hosts.

Reply:
I hear that. I guess I was wondering if millions of them could be a thing? Is there a limit?

Reply:
I'm not sure what the rationale upper-bound would be, though I assume at some point your local memory would fill up and crash. I think by saying "thousands", we're implying you have to deal with larger quantities too.